core/integration/storagetest/batch.go

```diff
@@ -116,7 +116,7 @@ func (*BatchTests) TestReadBatch(ctx context.Context, t *testing.T, f BatchStora
 }
 
 func (*BatchTests) TestHighestRev(ctx context.Context, t *testing.T, f BatchStorageFactory) {
-	domainID := "writebatchtest"
+	domainID := "writehighestrev"
```

Comment:
nit: Be consistent with naming these, e.g. use `"TestHighestRev"` as you do in other tests.